sovrin_node/server/node_authn.py

```diff
@@ -10,6 +10,7 @@ class NodeAuthNr(NaclAuthNr):
     def __init__(self, ledger: Ledger):
         self.ledger = ledger
 
+    # TODO: This is not right, what if verkey changes?
```

Comment:
It's very inefficient to get all transactions and iterate through them. Do we use this method? If we use it, then it needs to be fixed.